src/coffee/run.coffee

```diff
@@ -103,8 +119,8 @@ Promise.all [readCsvAsync(argv.types), readCsvAsync(argv.attributes)]
         process.exit 0
   .catch (e) ->
     console.error "Oops, something went wrong: #{e.message}"
-    process.exit 1
```

Comment:
why do remove the this? it is used to set the proper return value in case of a error